setup.py

```diff
@@ -1,18 +1,20 @@
 #!/usr/bin/env python
 
+import re
+
 from setuptools import setup
 from setuptools import find_packages
 from setuptools import Extension
-import re
 
+from mys.version import __version__
+
+
+with open('requirements.txt') as f:
+    requirements = [line for line in f]
```

Comment:
requirements.txt lists required packages for developing Mys. `install_requires` lists required packages to install Mys.